### PR TITLE
Fix:#86 added show details

### PR DESCRIPTION
**Pull request by ac491**
#86 added show details in PDF actions

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -28,6 +30,7 @@
         <item>rename</item>
         <item>print</item>
         <item>e-mail</item>
+        <item>details</item>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml]: Add these strings to other strings.xml also

File: app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java [hunk 2]
```diff
@@ -360,6 +371,32 @@ private void shareFile(String name) {
...
+        title.setPadding(20, 10, 10, 10);
+        title.setTextSize(30);
+        title.setTextColor(R.color.black_54);
+        new AlertDialog.Builder(mContext)
+                .setView(message)
+                .setCustomTitle(title)
+                .setPositiveButton("OK", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialogInterface, int i) {
+
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java]: dismiss the dialog on clicking ok button

File: app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java [hunk 3]
```diff
@@ -145,8 +149,11 @@ public void onBindViewHolder(@NonNull ViewFilesHolder holder, int position) {
         Log.d("logs", "getItemCount: " + mFileList.size());
         // Extract file name from path
         final String fileName = mFileList.get(position).getPath();
+        File file=mFileList.get(position);
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java]: space before and after =

**Comment by Swati4star** [comment]
Fix the checkstyle error.
And then squash the commits into a single commit.

**Comment by Swati4star** [comment]
@ac491 There are still 2 commits in this PR. Squash it into one single commit.
Changes look good 👍

**Comment by Swati4star** [comment]
But it is linked to PR now. If I merge your PR, it will be added to the commit history.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -69,10 +69,10 @@ Finally, go to GitHub and [make a Pull Request][] :D
 Travis CI and Circle CI will run our test suite. We care about quality, so 
 your PR won't be merged until all tests pass. 
 
-### 8. Stash the changes after review is done
+### 8. Squash the changes after review is done
```
Comment (Swati4star) [inline CONTRIBUTING.md]: This shouldn't be a part of your commit

**Comment by Swati4star** [comment]
@ac491 You PR has other's commits also. 
https://github.com/Swati4star/Images-to-PDF/commit/06e5bb6f99fda5122a4ce05578bce9f1217477e0
https://github.com/Swati4star/Images-to-PDF/commit/2e62480d71b9db96868f1b403097c5211407805f

File: CONTRIBUTING.md (hunk 4 above)
Comment (ac491) [reply CONTRIBUTING.md]: @Swati4star  how do I remove these commits?

Comment (Swati4star) [reply CONTRIBUTING.md]: You have squashed other's commits into your commit.
You can particularly pick your commit, and apply it to PR.
https://stackoverflow.com/questions/16931746/how-to-unsquash-a-commit

Comment (ac491) [reply CONTRIBUTING.md]: where can I find the commit reference of just my commit?
should I do git reset --hard HEAD^ and then commit the changes?

Comment (Swati4star) [reply CONTRIBUTING.md]: I don't know the series of command you exactly used to create this commit. 
You can unsquash your commits. Then, you'll get the reference of just your commit.

**Pull request closed without merging**